modular_skyrat/code/modules/projectiles/guns/energy/special.dm

```diff
@@ -0,0 +1,14 @@
+/obj/item/gun/energy/plasmacutter
+	sharpness = IS_SHARP_ACCURATE //can't sharpen, just to be sure.
+	toolspeed = 0.75 //plasmacutters can be used as welders, and are 25% faster than standard welders
```

Comment:
very small 5% increase from what it was before, but still undocumented

Reply:
I will document it. Good?

Reply:
yup thanks